packages/@react-aria/toast/src/useToast.ts

```diff
@@ -29,12 +29,15 @@ interface ToastAria {
   closeButtonProps: DOMProps & PressProps
 }
 
-export function useToast(props: AriaToastProps): ToastAria {
+export function useToast(props: ToastAriaProps): ToastAria {
```

Comment:
I just noticed that the props here contains the state stuff as well. Typically the other aria hooks have state as a separate input parameter for clarity purposes (see https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-aria/calendar/src/useCalendar.ts#L22 or https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-aria/calendar/src/useCalendar.ts#L22)